resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -60,1404 +60,1412 @@
+        - name: biomarkers__description_of_biomarkers
+          label: Biomarkers/Description
+          type: basic:string
+          required: false
+        - name: tme_panel_1__ann_call
+          label: TME Panel-1/ANN Call
+          type: basic:string
+          choices:
+            - label: A
+              value: a
+            - label: IA
+              value: ia
+            - label: ID
+              value: id
+            - label: IS
+              value: is
+            - label: N/A
+              value: na
+          description: |
+            Class type e.g. A for angio, IA for immune-angio, ID for immune-desert and IS for immune-suppressed.
+          required: false
+        - name: tme_panel_1__ann_prob
+          label: TME Panel-1/ANN Prob
+          type: basic:decimal
+          description: |
+            Probability score.
+          required: false
+        - name: tme_panel_1__ann_angio
+          label: TME Panel-1/ANN Angio
+          type: basic:decimal
+          description: |
+            Angio score.
+          required: false
+        - name: tme_panel_1__ann_immune
+          label: TME Panel-1/ANN Immune
+          type: basic:decimal
+          description: |
+            Immune score.
+          required: false
+        - name: tme_panel_1__ann_prob_a
+          label: TME Panel-1/ANN Prob A
+          type: basic:decimal
+          description: |
+            Probability angio.
+          required: false
+        - name: tme_panel_1__ann_prob_ia
+          label: TME Panel-1/ANN Prob IA
+          type: basic:decimal
+          description: |
+            Probability immune-angio.
+          required: false
+        - name: tme_panel_1__ann_prob_id
+          label: TME Panel-1/ANN Prob ID
+          type: basic:decimal
+          description: |
+            Probability immune-desert.
+          required: false
+        - name: tme_panel_1__ann_prob_is
+          label: TME Panel-1/ANN Prob IS
+          type: basic:decimal
+          description: |
+            Probability immune-suppressed.
+          required: false
+        - name: tme_panel_1__description_tme_panel_1
+          label: TME Panel-1/Description
+          type: basic:string
+          required: false
+        - name: signatures__immune_score
+          label: Signatures/Immune score
+          type: basic:decimal
+          description: |
+            Immune-classification of tumors by quantification of the density and location of immune-cells within the tumor.
+          required: false
+        - name: signatures__stromal_score
+          label: Signatures/Stromal score
+          type: basic:decimal
+          description: |
+            Stromal classification of tumors by quantification of stromal cells in TME.
+          required: false
+        - name: signatures__emt_score
+          label: Signatures/EMT score
+          type: basic:decimal
+          description: |
+            Epithelial–mesenchymal transition score.
```

Comment:
The character U+2013 "–" could be confused with the character U+002d "-", which is more common in source code.